### PR TITLE
Info and updates for 2016

### PR DESCRIPTION
**Pull request by asgrim**

### CODE REVIEW

File: information.html [hunk 1]
```diff
@@ -118,6 +139,12 @@
                 the ferry terminal is on the north-west of the city.
             </p>
 
+            <h4>By Hovercraft</h4>
+            <p>
+                Arrive in style from the Isle of Wight; check out <a href="http://www.hovertravel.co.uk/">the Hover
+                Travel</a> website for information about a speedy trip from IoW.
```
Comment (lboynton) [inline information.html:145]: > speedy trip from IoW

"speedy trip to the IoW"?

Comment (asgrim) [inline information.html:145]: No... it's information for getting _to_ the conf :dancer:

File: schedule.html [hunk 2]
```diff
@@ -456,7 +460,34 @@
...
+                    <p>
+                        Splice the Mainbrace! Delegates who have completed their treasure map will qualify for a
+                        traditional tot of rum, just like the Royal Navy. The rums to choose from will be:
+                        <ul>
+                            <li>Chairman's Reserve Spiced Rum</li>
+                            <li>Appleton Special Jamaican Rum</li>
+                            <li>Kraken Black Rum</li>
+                            <li>Captain Morgan's Spiced Rum</li>
+                            <li>Flora de Ca&ntilde;a 470 Extra Serco White Rum</li>
+                            <li>Lambs Navy Rum</li>
```
Comment (lboynton) [inline schedule.html]: Lamb's

File: schedule.html [hunk 3]
```diff
@@ -456,7 +460,34 @@
...
+                        </ul>
+                    </p>
+
+                    <strong>Free Bar</strong>
+                    <p>
+                        Everyone will get some tokens for our free bar, featuring a selection local ales:
```
Comment (lboynton) [inline schedule.html]: > selection local ales

selection of local ales

File: schedule.html [hunk 4]
```diff
@@ -456,7 +460,34 @@
...
+                        <ul>
+                            <li>Mary Rose Ale</li>
+                            <li>Nuptu'Ale</li>
+                            <li>Pompey Royal</li>
+                            <li>Hole Hearted</li>
+                            <li>Ringwood Ale</li>
+                        </ul>
+                        There will also be a selection of beers, wines, and soft drinks.
```
Comment (lboynton) [inline schedule.html]: "a selection _other_ beers"?

File: information.html (hunk 1 above)
Comment (lboynton) [inline information.html:145]: Er yes, of course

Comment (asgrim) [inline information.html:145]: If you live on IoW, it might be useful :P

Comment (lboynton) [inline information.html:145]: I was confused because I'm on the IoW right now, and thought the conference was here for a moment

**Comment by asgrim** [comment]
@lboynton updated everything else

**Comment by asgrim** [comment]
Thanks!

**Pull request merged**